channels/viber/src/main/kotlin/com/justai/jaicf/channel/viber/sdk/Jackson.kt

```diff
@@ -0,0 +1,18 @@
+package com.justai.jaicf.channel.viber.sdk
```

Comment:
This file should have proper JvmName for java clients. Otherwise generated name will be JacksonKt, which is a bad name for public class